src/main/kotlin/io/titandata/titan/providers/local/Delete.kt

```diff
@@ -13,4 +13,26 @@ class Delete (
         commitsApi.deleteCommit(repository, commit)
         println("$commit deleted")
     }
+
+    fun deleteTags(repository: String, commit: String, tags: List<String>) {
+        val commit = commitsApi.getCommit(repository, commit)
+        val commitTags = (commit.properties.get("tags") as Map<String, String>).toMutableMap()
+
+        for (t in tags) {
+            if (t.contains("=")) {
+                val key = t.substringBefore("=")
```

Comment:
Is there any validation for a string that contains more than 1 = ?

Reply:
No, because that's totally legal. You can definitely do "foo=bar=baz". That will equate to ("foo", "bar=baz"). Weird, but that's why I use subStringBefore and not substringBeforeLast.